core/types/lending_transaction.go

```diff
@@ -43,6 +43,8 @@ const (
 	LendingStatusCancelled     = "CANCELLED"
 	LendingTypeMo              = "MO"
 	LendingTypeLo              = "LO"
+	LendingStatusPayment       = "PAYMENT"
```

Comment:
This status is renamed to `REPAY` https://github.com/tomochain/tomochain/blob/tomoxlending/tomoxlending/lendingstate/lendingitem.go#L19